charts/azuremonitor-containers/templates/ama-logs-deployment.yaml

```diff
@@ -35,13 +35,44 @@ spec:
    containers:
 {{- if and (ne .Values.Azure.Cluster.ResourceId "<your_cluster_id>") (.Values.amalogs.useAADAuth) }}
      - name: addon-token-adapter
+       # Extension specifies default, platform overrides during deployment if needed
+       image: "mcr.microsoft.com/aks/msi/addon-token-adapter:master.250604.1"
```

Comment:
same here . instead of making changes to existing yaml snippet,  we should add new yaml snippet if its openshift.

Reply:
changed